custom_components/powercalc/config_flow.py

```diff
@@ -2050,11 +2064,20 @@ def build_basic_options_schema(self) -> vol.Schema:
                 },
             )
 
+        schema = vol.Schema(
+            {
+                **SCHEMA_ENERGY_SENSOR_TOGGLE.schema,
+                **SCHEMA_UTILITY_METER_TOGGLE.schema,
+            },
+        )
+        if not (self.selected_profile and self.selected_profile.only_self_usage):
```

Comment:
Seems like this removes the standby power option for all devices without a profile (manually configured), maybe update the conditional to something like:
```suggestion
if (not self.selected_profile) or (not self.selected_profile.only_self_usage):
```